source/features/clear-pr-merge-commit-message.tsx

```diff
@@ -15,6 +16,18 @@ function init(): void | false {
 		deduplicatedAuthors.add('Co-authored-by: ' + author);
 	}
 
+	// Preserve closing issues numbers when a pr is merged into a non-default branch since GitHub doesn't close them #4531
+	const baseBranch = select('.base-ref a')!.title.split(':')[1];
```

Comment:
I think we have (or used to have) a helper function for this.

Reply:
Long ago but I found another one to use